dojo/finding/views.py

```diff
@@ -1126,15 +1126,6 @@ def edit_finding(request, fid):
                 if jform.is_valid():
                     push_to_jira = jform.cleaned_data.get('push_to_jira')
 
-                    if JIRA_Issue.objects.filter(finding=new_finding).exists():
```

Comment:
The reason I had removed it from `models.py` if my other PR, is because the GitHub integration right below remains. So in the end, this `views.py` remains unclean.